Add instruction text and Dashboard

manu29d opened the pull request:
<img width="927" alt="screenshot 2018-12-26 at 4 40 24 pm" src="https://user-images.githubusercontent.com/3751990/50449301-6f3ddb00-094c-11e9-9dd6-6354ff18b1c9.png">

manu-curve committed:
Add instruction text

sushantahujaa commented on src/dashboard/Dashboard.js:
```diff
@@ -0,0 +1,17 @@
+  render() {
+    return (
+      <div className='Dashboard'>
+        <h1>Quizapur</h1>
```
we need to change the instructions text

manu29d replied:
But should we?

Commit:
Adds React - BrowserRouter, Route, Link

Commit:
Fix routing issue

Story: New component was being rendered below the previous one
It should replace the old component

Bug: <Route /> was added to <Dashboard />

Fix: Move <Route /> to <App />

Commit:
Rename startQuiz.js to StartQuizBtn.js

Commit:
Rename instructions.js to Instructions.js

Commit:
Removes unused variable - Link

mounika-puppala committed:
Merge pull request #3 from sushantahujaa/mp-startQuiz-btn

Added Start Quiz button to dashboard page
